app/controllers/tests_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TestsController < ApplicationController
+  def index
+    render "tests/index"
```

Comment:
I had to explicitly render here which I'm generally a fan of, but for seamless integration between view_component and rails I think we'd need to figure out how to use implicit rendering too.